raft/src/main/java/org/apache/kafka/raft/internals/ExternalKRaftMetricIgnoredStaticVoters.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.raft.internals;
+
+public interface ExternalKRaftMetricIgnoredStaticVoters {
+    void switchIgnoredStaticVoters();
```

Comment:
I know that I suggested using the "switch" prefix but I am starting to think that it is not fully accurate. Maybe the prefix "set" is the best option.